scripts/market-wfrp4e.js

```diff
@@ -436,7 +428,7 @@ class MarketWfrp4e {
         } else //generate a card with a summary and a receive button
         {
             let amount
-            let nbActivePlayers = getNbOfActivePlayers();
+            let nbActivePlayers = game.users.players.filter(p => p.data.active).length;            
```

Comment:
How does this fix anything? It's the same line of code.

Reply:
Update.... This is the proper fix ...